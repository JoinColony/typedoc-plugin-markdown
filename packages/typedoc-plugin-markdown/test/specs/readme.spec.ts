import * as Handlebars from 'handlebars/runtime';
import { ProjectReflection } from 'typedoc';
let project: ProjectReflection;

describe(`Readme:`, () => {
  let readmeTemplate: Handlebars.TemplateDelegate;

  beforeAll(async () => {
    const bootstrap = global.bootstrap('reflections.ts', {
      stubHelpers: ['breadcrumbs'],
    });
    project = bootstrap.project;
    readmeTemplate = global.getTemplate(bootstrap.context, 'readme', false);
  });

  test(`should compile readme`, () => {
    expect(
      global.renderTemplate(readmeTemplate, {
        model: project,
        project: project,
      }),
    ).toMatchSnapshot();
  });
});
