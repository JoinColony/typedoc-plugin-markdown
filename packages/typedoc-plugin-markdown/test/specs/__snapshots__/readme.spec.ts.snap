// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Readme: should compile readme 1`] = `
"[helper: breadcrumbs]

# MY README

> Some block quote

<!-- some comments -->

Some text
"
`;
